mealie/core/security/security.py

```diff
@@ -22,11 +22,10 @@ class UserLockedOut(Exception):
     ...
 
 
-def create_access_token(data: dict, expires_delta: timedelta | None = None) -> str:
+def create_access_token(data: dict, expires_delta: timedelta) -> str:
     settings = get_app_settings()
 
     to_encode = data.copy()
-    expires_delta = expires_delta or timedelta(hours=settings.TOKEN_TIME)
```

Comment:
I think if you leave this method alone it should achieve the same thing you're looking for without breaking other functionality, since in the instance you fixed it always passes an expires_delta